static/app/components/dateTime.tsx

```diff
@@ -84,8 +85,8 @@ class DateTime extends React.Component<Props> {
     return (
       <time {...carriedProps}>
         {utc
-          ? moment.utc(date).format(format)
-          : moment.tz(date, options?.timezone ?? '').format(format)}
+          ? moment.utc(date as moment.MomentInput).format(format)
```

Comment:
Is this a safe cast?

Reply:
The moment type file changed from `null | undefined` to `void` ([see commit](https://github.com/moment/moment/commit/5a1843de5a98025aa04437df8d7e4e7200306704)), which is more accurate. It'll work just fine with the casting.